specifications/passive/dreem-1.0.0.yml

```diff
@@ -0,0 +1,21 @@
+#====================================== For DREEM device =====================================#
+vendor: DREEM
+model: NA
+version: 1.0.0
+doc: DREEM data collected and uploaded by DREEM web application
+data:
+  - type: DREEM SLEEP STAGE
+    unit: NON_DIMENSIONAL
+    processing_state: VENDOR
+    topic: dreem_sleep_stage
+    value_schema: .passive.dreem.DreemSleepStage
+  - type: DREEM SLEEP ENDPOINTS
+    unit: NON_DIMENSIONAL
+    processing_state: VENDOR
+    topic: dreem_sleep_endpoints
+    value_schema: .passive.dreem.DreemSleepEndpoints
+  - type: DREEM METRIC
```

Comment:
I think there are only two data topics. One is sleep endpoints and the other is the sleep stage
The Dreem metric is used within the sleep endpoints.